Before: 62489c438f659f4175dbbb3ee4bd753239e683ac0585c155af01d1a99ae97247
After: 8c6e5ef21ac9c40912aba7ba2fe7824aeaa3707024e0495fed9005ad55ede9f1
Patch: 6e76ce050e4d384a0dec7d15f54c9dd4e7cf5f52d7e903df68560bc1f6187398
--- .yardopts	2012-07-22 20:44:49.315583788 +1000
+++ .yardopts	2012-07-22 21:28:15.316780245 +1000
@@ -2,5 +2,4 @@
 --output-dir target/doc
 --markup     markdown
 --readme     README.md
---files      CHANGELOG.md
 {app,lib}/**/*.rb
--- CHANGELOG.md	2012-07-22 20:44:49.315583788 +1000
+++ /dev/null	2012-07-22 20:29:06.375709578 +1000
@@ -1,6 +0,0 @@
-My Project: Changelog
-=====================
-
-## 0.0.1 (WIP)
-
-Initial version.
